feature/home/src/commonMain/kotlin/com/chesire/nekomp/feature/home/ui/HomeViewModel.kt

```diff
@@ -78,18 +79,26 @@ class HomeViewModel(
         )
     }
     private val _airingSeries = showAiringSeries().map { it.toPersistentList() }
+    private val _pendingUpdates = MutableStateFlow<List<Int>>(emptyList())
```

Comment:
This is horrible, but I can't immediately think of a better way with how the UIState is built so 🤷